app/views/products/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<main class="row">
+
+
+
+<h3 class="centering">Ghosty Products for ghosty needs</h3>
+
+<h3 class="columns large-12 small-12"><%= @product_filter %></h3>
```

Comment:
If you have the diff sizes with the same values its not going to do you much good